loopchain/channel/channel_service.py

```diff
@@ -466,8 +466,8 @@ def __ready_to_height_sync(self):
             if self._is_genesis_node():
                 self.generate_genesis_block()
 
-        if not self.is_support_node_function(conf.NodeFunction.Vote) and not ChannelProperty().rs_target:
-            utils.exit_and_msg(f"There's no radiostation target to sync block.")
+        # if not self.is_support_node_function(conf.NodeFunction.Vote) and not ChannelProperty().rs_target:
+        #     utils.exit_and_msg(f"There's no radiostation target to sync block.")
```

Comment:
Why don't you remove if this lines useless? or add short description why you add comment this lines.

Reply:
The PR is for operating loopchain as a single node. So RS Target is not required. But current loopchain requires RS Target. 
The reason why I leave it as a commit is We do not decide how we solve it yet.